stable/jenkins/templates/jenkins-agent-svc.yaml

```diff
@@ -25,3 +25,6 @@ spec:
     "app.kubernetes.io/component": "{{ .Values.master.componentName }}"
     "app.kubernetes.io/instance": "{{ .Release.Name }}"
   type: {{ .Values.master.slaveListenerServiceType }}
+  {{if eq .Values.master.slaveListenerServiceType "LoadBalancer"}}
```

Comment:
Should the If condition also check if an IP is set?

Reply:
Totally agree. Gonna do it right away.

Reply:
Done